Source/sound.cpp

```diff
@@ -240,11 +256,10 @@ void music_start(uint8_t nTrack)
 				return;
 			}
 
-			Mix_VolumeMusic(MIX_MAX_VOLUME - MIX_MAX_VOLUME * sgOptions.Audio.nMusicVolume / VOLUME_MIN);
-			if (Mix_PlayMusic(music, -1) < 0) {
-				SDL_Log("Mix_PlayMusic: %s", Mix_GetError());
-				Mix_FreeMusic(music);
-				music = nullptr;
+			music->setVolume(1.0F - 1.0F * sgOptions.Audio.nMusicVolume / VOLUME_MIN);

```

Comment:
static_cast<float>? Another one a few lines below.

Writing of float literals seems to randomly switch between using uppercase F and lowercase f, and just a '.' or '.0'